platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -3890,7 +3937,7 @@
 
               $http.get('../mds/entities/'+$scope.selectedEntity.id+'/entityFields').success(function (data) {
                    $scope.allEntityFields = data;
-                   $scope.setAvailableFieldsForDisplay();
+                   $scope.setAvailableFieldsForDisplay($scope);
```

Comment:
What's the point of passing scope as var? Are we doing recursion here? I think I missed it. Or is this called from somewhere else with a different scope?